com.unity.render-pipelines.universal/Runtime/PostProcessPasses.cs

```diff
@@ -30,28 +45,40 @@ internal struct PostProcessPasses : IDisposable
 
         public bool isCreated { get => m_CurrentPostProcessData != null; }
 
-        public PostProcessPasses(PostProcessData rendererPostProcessData, Material blitMaterial)
+        private PostProcessPasses(PostProcessData rendererPostProcessData)
         {
             m_ColorGradingLutPass = null;
             m_PostProcessPass = null;
             m_FinalPostProcessPass = null;
             m_CurrentPostProcessData = null;
+            m_BlitMaterial = null;
 
             m_AfterPostProcessColor = null;
             m_ColorGradingLut = null;
-
             m_RendererPostProcessData = rendererPostProcessData;
-            m_BlitMaterial = blitMaterial;
+        }
 
+        public PostProcessPasses(PostProcessData rendererPostProcessData, ref PostProcessParams postProcessParams) : this(rendererPostProcessData)
+        {
+            m_BlitMaterial = postProcessParams.blitMaterial;
+            Recreate(rendererPostProcessData, ref postProcessParams);
+        }
 
-            Recreate(rendererPostProcessData);
+        [Obsolete]
```

Comment:
Same as the other class, this one is in internal namespace so you can just remove the constructor.

Reply:
Yeah. I guess I've originally forgotten to look at the class visibility then.

Reply:
fixed.